src/types/index.ts

```diff
@@ -137,6 +137,12 @@ export interface PeriodGeoZoneTicket extends BasePeriodTicket {
     stops: Stop[];
 }
 
+export interface MultiOperatorGeoZoneTicket extends PeriodGeoZoneTicket {
```

Comment:
We already have this as an optional thing in the base ticket, whats the reasoning on adding this?